PublicationsRouter/v2/api/API.md

```diff
@@ -302,7 +392,7 @@ Here **notification_id** is the system's identifier for an individual notificati
 
 If the notification does not exist, you will receive a 404 (Not Found), and no response body.
 
-If the you are not authenticated as the original publisher of the notification, and the notification has not yet been routed, you will also receive a 404 (Not Found) and no response body.
+If the you are not authenticated as the original publisher of the notification, and the notification has not yet been routed, you will also receive a 404 (Not Found) and html error page in the body.
```

Comment:
since we created the Jisc error pages the endpoints return the 404.html 
Shall I replace it in all the documentation?